src/vis/bar/BarVis.tsx

```diff
@@ -113,11 +112,11 @@ export function BarVis({
         const menu = document.getElementById(`generalVisBurgerMenu${uniqueId}`);
 
         menu.addEventListener('hidden.bs.collapse', () => {
-            Plotly.Plots.resize(document.getElementById(`plotlyDiv${uniqueId}`));
+            window.dispatchEvent(new Event('resize'));
```

Comment:
Why did you change this resize event to be dispatched on the `window`? This might cause rerenders and other changes on different parts of the application. Example: Take two general vis panels in two different views/workbenches in Ordino, toggling the vis config panel will also re-render the other general vis which must not be resized. I would revert this change, unless there are other reasons that I currently don't see.